lib/node_modules/@stdlib/blas/ext/base/ndarray/dapx/docs/types/index.d.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { float64ndarray } from '@stdlib/types/ndarray';
+
+/**
+* Adds a scalar constant to each element in a double-precision floating-point ndarray.
+*
+* @param arrays - array-like object containing an input ndarray and a zero-dimensional ndarray containing the scalar constant
+* @returns input ndarray
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+* var ndarray = require( '@stdlib/ndarray/ctor' );
+* var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+*
+* var xbuf = new Float64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+* var x = new ndarray( 'float64', xbuf, [ 4 ], [ 1 ], 0, 'row-major' );
+*
+* var alpha = scalar2ndarray( 5.0, {
+*     'dtype': 'float64'
+* });
+*
+* var out = dapx( [ x, alpha ] );
+* // returns <ndarray>
+*/
+declare function dapx( arrays: ArrayLike<float64ndarray> ): float64ndarray;
```

Comment:
```suggestion
declare function dapx( arrays: [ float64ndarray, float64ndarray ] ): float64ndarray;
```

By using `ArrayLike`, you end up losing type specificity. Here, we can specify that one must provide exactly two ndarrays.